lms/djangoapps/support/static/support/jsx/entitlements/data/reducers/error.js

```diff
@@ -0,0 +1,14 @@
+import { errorActions } from '../actions/constants';
+
+const error = (state = '', action) => {
+  switch(action.type){
+    case errorActions.DISPLAY_ERROR:
```

Comment:
Do you need to clear the error state if a request initially failed but then subsequently succeeded? 

I think currently, the failure of the first request would mean that `state.error` is `DISPLAY_ERROR` and then the success of the second request would mean that `state.entitlements` is successfully defined but `state.error` is still `DISPLAY_ERROR` when it should really be empty or some other `falsy` state.

Reply:
Depends on what the behavior we want is. Currently I thought it was somewhat sensible to just show the error until it was dismissed, I suppose it would also make sense to dismiss it when a call went through though.

Reply:
Good point though, I've added to the error reducer to update on successful fetch.